lib/tasks/goodcity/packages_locations/delete_dispatched_locations.rake

```diff
@@ -0,0 +1,12 @@
+namespace :goodcity do
+  namespace :packages_locations do
+
+    desc "Delete dispatched packages_locations"
+    task delete_dispatched_locations: :environment do
+      # Use delete instead of destroy so we don't fire push updates
+      location_id = Location.dispatch_location.id
+      PackagesLocation.where(location_id: location_id).delete_all
```

Comment:
note: we should only be deleting those that have been fully dispatched

Reply:
Yes, we are using `delete_all` in order to avoid triggering `push_updates`.

When we run this rake task, there should be no case of partially dispatched packages as the UI doesn't allow it. So it should be safe.